torch/jit/__init__.py

```diff
@@ -1741,7 +1741,7 @@ def __contains__(self, key):
         # dir is defined by the base nn.Module, so instead of throwing if
         # it is not overriden, we call into the nn.Module __dir__ method
         def __dir__(self):
-            self_method = getattr(self, "__dir__")
+            self_method = self.__dir__
```

Comment:
Was there a good reason to do this? Why was getattr used to begin with?

Reply:
I don't think so? They are basically the same when used with a string literal, so I guess it was just reflex.

Reply:
Ok, I'm not sure why you would ever use `getattr` with constant strings. That's why I was asking.
But the fix looks better yes.